### PR TITLE
Inputs Outputs Content Updates v1.1

### PR DESCRIPTION
**Pull request by toriancrane**
## Description

This PR provides the following content updates based on feedback from the previous release:

- Moves the outputs "helper functions" to live underneath apply/all respectively
- Updates the interpolation content and code examples on the all page
- Additional content/verbiage changes and updates

### CODE REVIEW

**Review by scottslowe — COMMENTED** [review]
This all looks good; I really like the new arrangement/layout of the information and the way it's presented.

The only thing preventing an approval is this: are all the `go.mod.txt` changes intentional? I only ask because I've had some PRs where those files were modified and the changes weren't intentional/desired.

File: themes/default/content/docs/concepts/inputs-outputs/apply.md [hunk 1]
```diff
@@ -815,6 +1118,557 @@ Duration: 5s
...
+}));
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+### Parsing JSON Outputs to Objects
+
+You can parse a JSON string into an object and then, inside of an Apply, manipulate the object to remove all of the policy statements.
```
Comment (cnunciato) [inline themes/default/content/docs/concepts/inputs-outputs/apply.md:1484]:
```suggestion
You can parse a JSON string into an object and then, inside of an `apply`, manipulate the object to remove all of the policy statements:
```

**Review by cnunciato — APPROVED** [review]
Looks _awesome_. ✨  A few minor suggestions for you to consider, but nothing need block. 🚀

**Comment by torian-qlc** [comment]
> This all looks good; I really like the new arrangement/layout of the information and the way it's presented.
> 
> The only thing preventing an approval is this: are all the `go.mod.txt` changes intentional? I only ask because I've had some PRs where those files were modified and the changes weren't intentional/desired.

Good catch! Definitely not intentional. Do you know why this happens?

**Comment by scottslowe** [comment]
> > This all looks good; I really like the new arrangement/layout of the information and the way it's presented.
> > The only thing preventing an approval is this: are all the `go.mod.txt` changes intentional? I only ask because I've had some PRs where those files were modified and the changes weren't intentional/desired.
> 
> Good catch! Definitely not intentional. Do you know why this happens?

@toriancrane Honestly, I have no idea. I've had it happen to me and Christian has helped me revert the changes, but we have not yet been able to determine exactly _why_ it happens. I think it happened to Zack once, too, before he left. It's something to do with the pre-commit script, I think.

**Pull request merged**